sql/connect/common/src/main/protobuf/spark/connect/expressions.proto

```diff
@@ -457,3 +458,17 @@ message LazyExpression {
   // (Required) The expression to be marked as lazy.
   Expression child = 1;
 }
+
+message SubqueryExpression {
+  // (Required) The id of corresponding connect plan.
+  int64 plan_id = 1;
+
+  // (Required) The type of the subquery.
+  SubqueryType subquery_type = 2;
+
+  enum SubqueryType {
+    SUBQUERY_TYPE_INVALID = 0;
```

Comment:
I'd probably use UNKNOWN rather than INVALID. The only reason we add this first enum element is because proto defaults to the first element if you do not specify a value.